### PR TITLE
Unicode emoji escape

### PR DESCRIPTION
**Pull request by kistitan**
Workaround until server support

### CODE REVIEW

File: FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java [hunk 1]
```diff
@@ -0,0 +1,1075 @@
...
+            emojiUnicodes[i] = new String(Character.toChars(key));
+            emojiEscapes[i] = emojisMap.get(key).toString();
+        }
+    }
+
+    //escapes should be longer than 2 characters
+    static void initEmojiMap() {
+
+        // People
+        emojisMap.put(0x1f604, ":emoji_1f604:");
```
Comment (jacortinas) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: @kistitan Does this mean the server is receiving `:emoji_#####` text in messages?

Comment (kistitan) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: yes, better idea?

Comment (jacortinas) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: I agree we should be supporting emoji 😬

Comment (jacortinas) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: I don't have any better ideas, but we need to synchronize with the server side to make sure that user's aren't seeing `:emoji_#####` text on the web in random places. There are libraries that let us translate from unicode emoji to the ASCII text like `:smile:` that basically do what you're doing but do in on the server.

@reneklacan Check this out.

Comment (kistitan) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: I can add also built in java escape for the unicode characters, but in this case the user will see just some strange characters after each other.
Or would be easier for you to hide?

Comment (kistitan) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: Is it a very nasty thing if the user sees escaped emojis in his/her text?

Comment (jacortinas) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: @kistitan I think so 😬

Comment (kistitan) [inline FetLife/fetlife/src/main/java/com/bitlove/fetlife/util/EmojiUtil.java]: Hahaha ok :)
makes sense.
Useful review than.

**Pull request closed without merging**